build.sbt

```diff
@@ -21,7 +21,8 @@ libraryDependencies ++= Seq (
   scalactic
 ) ++ Seq(
   akkaHttpTestkit, 
-  scalatest, 
+  scalatest,
+  mockito,
```

Comment:
I did tried with ScalaMock, but with very limited success. I had an issue with curried functions where Mockito does the job better in a more readable way.